test/e2e/utils_test.go

```diff
@@ -161,12 +161,12 @@ func cleanupCRP(name string) {
 		if errors.IsNotFound(err) {
 			return nil
 		}
-		Expect(err).Should(Succeed(), "Failed to get CRP %s", name)
+		g.Expect(err).Should(Succeed(), "Failed to get CRP %s", name)
 
 		// Delete the CRP (again, if applicable).
 		//
-		// This helps the AfterAll node to run successfully even if the steps above fail early.
-		Expect(hubClient.Delete(ctx, crp)).To(Succeed(), "Failed to delete CRP %s", name)
+		// This helps the After All node to run successfully even if the steps above fail early.
+		g.Expect(hubClient.Delete(ctx, crp)).To(Succeed(), "Failed to delete CRP %s", name)
```

Comment:
What is the difference between this and before?

Reply:
We can use expect statements with Eventually blocks but only if we use the gomega object which intercepts the failures correctly and fails if any one of the Expect statements within Eventually block fails https://onsi.github.io/gomega/#category-3-making-assertions-eminem-the-function-passed-into-codeeventuallycode